ateam_vision_filter/src/camera.hpp

```diff
@@ -100,6 +107,7 @@ class Camera
   std::array<MultipleHypothesisTracker, 16> yellow_team;
   std::array<MultipleHypothesisTracker, 16> blue_team;
   MultipleHypothesisTracker ball;
+  int ignore_half_ = 0;
```

Comment:
Might fix this in the future, but it currently matches a bunch of other code in other packages because this same convention is used in the ROS message.